packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -68,6 +74,12 @@ const useBarStyles = makeStyles({
       backgroundColor: 'Highlight',
     },
   },
+  rounded: {
+    ...shorthands.borderRadius(tokens.borderRadiusMedium),
+  },
+  rectangular: {
+    ...shorthands.borderRadius(tokens.borderRadiusNone),
+  },
```

Comment:
nit: you could use `inherit` instead:
```suggestion
    ...shorthands.borderRadius('inherit'),
  },
```